src/main/java/me/ryanhamshire/GriefPrevention/BlockEventHandler.java

```diff
@@ -232,6 +233,27 @@ public void onBlockPlace(BlockPlaceEvent placeEvent)
 		String noBuildReason = GriefPrevention.instance.allowBuild(player, block.getLocation(), block.getType());
 		if(noBuildReason != null)
 		{
+			// Allow players with container trust to place books in lecterns
+			PlayerData playerData = this.dataStore.getPlayerData(player.getUniqueId());
+			Claim claim = this.dataStore.getClaimAt(block.getLocation(), true, playerData.lastClaim);
+			if (block.getType() == Material.LECTERN && placeEvent.getBlockReplacedState() instanceof Lectern)
+			{
+				if (claim != null)
+				{
+					playerData.lastClaim = claim;
+					String noContainerReason = claim.allowContainers(player);
+					if (noContainerReason != null)
+					{
+						placeEvent.setCancelled(true);
+						GriefPrevention.sendMessage(player, TextMode.Err, noContainerReason);
+						return;
+					}
+					else
+					{
+						return;
+					}
```

Comment:
would suggest reducing code via

```java
if (noContainerReason == null)
    return;
// else logic, but now no longer needs `else`
```

Reply:
Good call, that will clean that up quite a bit.